mp/src/game/shared/momentum/mom_gamemovement.h

```diff
@@ -8,21 +8,9 @@
 
 class CMomentumPlayer;
 
-#define NO_REFL_NORMAL_CHANGE -2.0f
-#define BHOP_DELAY_TIME 15 // Time to delay successive bhops by, in ticks
 #define STOP_EPSILON 0.1
 #define MAX_CLIP_PLANES 5
```

Comment:
You can move these to the cpp as well if they aren't needed for anything inside this header / other files.

Reply:
Oh yeah, I just saw they also existed in other files, but those are cpp too, so I'll do that.

Reply:
STOP_EPSILON and MAX_CLIP_PLANES also exist in gamemovement.cpp and physics_main.cpp with the same values.

Reply:
Resolved in commit 67f4c92.